### PR TITLE
Add Supports to System Message to Language Model

### PR DESCRIPTION
**Pull request by papayalabs**
Some models do not support System Message. In Language Model is created a boolean that gives this information.

### CODE REVIEW

File: app/models/language_model.rb [hunk 1]
```diff
@@ -52,6 +52,10 @@ def supports_tools?
       api_service.name != "Groq" # TODO: Remove this short circuit once I can debug tool use with Groq
   end
 
+  def supports_system_message?
+    attributes["supports_system_message"]
+  end
```
Comment (krschacht) [inline app/models/language_model.rb:57]: You can cut this, the method is automatically added to all booleans

**Review by krschacht — CHANGES_REQUESTED** [review]
This looks good! Just a test, one comment, and also you forgot to commit the schema file. 

Oh, and update again from main.

**Review by krschacht — APPROVED** [review]
Looks great! I made two minor copy changes. As soon as the tests pass, I'll merge this in. Thanks for tackling this one!

**Comment by krschacht** [comment]
@papayalabs can you look at this one failing test? I don't see an obvious problem with the code or the test

**Comment by papayalabs** [comment]
@krschacht Hey!

I wrote this 2 tests on RegisterableTest:

`  test "it creates some language models with supports_system_message enabled and some without" do
    people(:ali_invited).update!(personable_type: "User", personable_attributes: { first_name: "John", last_name: "Doe", })
    assert people(:ali_invited).user.language_models.where(supports_system_message: false).present?
    assert people(:ali_invited).user.language_models.where(supports_system_message: true).present?
  end

  test "make sure gpt-best(first language model) has supports_system_message: true" do
    people(:ali_invited).update!(personable_type: "User", personable_attributes: { first_name: "John", last_name: "Doe", })
    assert_equal people(:ali_invited).user.language_models.first.supports_system_message, true
  end`

And supports_system_message is true but when reach the test (edit should have supports_system_message checkbox checked) comes as false.

**Comment by papayalabs** [comment]
Hey @krschacht! Now all tests pass.

**Comment by krschacht** [comment]
All looks good, merging now., sorry for being MIA a couple days.

**Pull request merged**